lib/api_repo.js

```diff
@@ -407,7 +407,9 @@ ApiRepo.prototype._buildProtos =
         }
       }
       if (language === 'java') {
-        var dstResourceDir = path.join(langTopDir, 'src', 'main', 'resources');
+        var baseDir = this.opts.altJava ? 'proto' : 'resources';
```

Comment:
Just to be clear: so in altJava situation, there are no such non-proto resources?